admin/static/js/sales_analytics/sa-products-view.js

```diff
@@ -0,0 +1,138 @@
+'use strict';
+
+var keen = require('keen-js');
+var ss = require('simple-statistics');
+
+var SalesAnalytics = function() {
+    var self = this;
+
+    self.keenClient = new keen({
+        projectId: keenProjectId,
+        readKey : keenReadKey
+    });
+
+    self.keenFilters = {
+        nullUserFilter: {
+            property_name: 'user.id',
+            operator: 'ne',
+            property_value: null
+        },
+        inactiveUserFilter: {
+            property_name: 'user.id',
+            operator: 'ne',
+            property_value: ''
+        }
+    };
```

Comment:
Yes lots of repetitions here and there. all the js files other than dashboard.js should be deleted in this case.